inst/workflow/3_reporting/Changes.yaml

```diff
@@ -0,0 +1,25 @@
+meta:
+  Type: Reporting
+  ID: Changes
+  Description: Calculate change from previous snapshot.
+  Priority: 2
+spec:
+  Reporting_Results:
```

Comment:
This all certainly makes sense to me, I think the only thing I'd have to say is when we run `BindResults()` via `Results.yaml` we are generally grabbing the `Sys.Date()` so singular results dataset tend to only have one snapshot at a time. The features/code from this PR seem aimed directly for that `Reporting_Results_Longitudinal` object we may have in our snapshotting process(?)

Reply:
Yeah it's a valid point... But our workflows are all set up in the context of a single snapshot, so it's up to the end user to implement in the context of multiple snapshots.  And I'm not sure we want to introduce the complexity of a longitudinal implementation here...